src/components/pages/SingleAccommodation.js

```diff
@@ -63,9 +73,103 @@ export class SingleAccommodation extends React.Component {
     await likeUnlikeAccommodation(accommodation.slug, 'unlike');
   }
 
+  handleChange = async (e) => {
```

Comment:
To avoid congesting this file, create a `Rating component` with all the rating logic and call it in this file.